includes/episode_number_column.php

```diff
@@ -25,3 +29,61 @@ function podlove_add_episodeno_column_content_to_episodes_table($column_name)
         }
 	}
 }
+
+function podlove_episodeno_quickedit_form($column_name)
+{
+    if ($column_name === 'episode_number') {
+			?>
+			<fieldset class="inline-edit-col-left">
+		    <div class="inline-edit-col">
+					<label class="alignleft">
+						<span class="title"><?php _e('Episode number', 'podlove-podcasting-plugin-for-wordpress') ?></span>
+						<input type="number" name="_podlove_meta[number]" class="podlove_meta_quickedit_episode_number" />
+					</label>
+					<?php wp_nonce_field( '_podlove_meta_quickedit_episode_number', '_podlove_meta_quickedit_episode_number_nonce_field' ); ?>
+		    </div>
+	    </fieldset>
+	    <?php
+		}
+}
+
+function podlove_episodeno_quickedit_save($post_id)
+{
+	if ( wp_verify_nonce($_POST['_podlove_meta_quickedit_episode_number_nonce_field'], '_podlove_meta_quickedit_episode_number') ) {
+		$episode = \Podlove\Model\Episode::find_one_by_post_id($post_id);
+		$episode->number = sanitize_text_field($_POST['_podlove_meta']['number']);
+
+		if ( is_string($episode->number) )
+			$episode->save();
+	}
+}
+
+function podlove_episodeno_quickedit_populate_form()
+{
+	global $current_screen;
+
+  if ($current_screen->post_type !== 'podcast') {
+  	return;
+  }
+
+	?>
+	<script type="text/javascript">
+		function podlove_update_quickedit_episode_number(number) {
+			jQuery('.podlove_meta_quickedit_episode_number').val(number);
+		}
+	</script>
+	<?php
+}
+
+function podlove_episodeno_quickedit_extend_action_items($actions, $post)
+{
+  if ($post->post_type !== 'podcast') {
+		return $actions;
+  }
+
+	$episode = \Podlove\Model\Episode::find_or_create_by_post_id($post->ID);
+	// Not nice but seems like there is in a more elegant way to do this right now
```

Comment:
Oh dear :) I would really prefer a solution without sneaking an onclick handler into the button. Have you seen https://wordpress.stackexchange.com/a/8098/743? Looks like an alternative approach.

Reply:
Hmm… of course we could (or should) make use of the jQuery on('click', …) event here but in the end we have to use some PHP zu provide the episode number to the JS. If you prefer this we could make use of a "data" attribute here.

However, my real problem with both solutions is that they rely on a specific class of a link/button or a string to add the onclick handler. If WP decides to make any changes to the class/string etc. the whole thing breaks.

Reply:
Put value via data attribute + jQuery.live(... seems an ok approach to me. 

_some_ reliance on WP stuff is unfortunate but seems unavoidable.